src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/incidents/IncidentService.kt

```diff
@@ -205,7 +205,7 @@ private fun IncidentRequirement.toRequirement() = Requirement(
   staff = recordingStaff.toStaff(),
   sequence = id.requirementSequence,
   comment = comment,
-  date = recordedDate,
+  date = recordedDate.toLocalDate(),
```

Comment:
ideally we'd pass the date time back, but that would change the contract.  So for now, just leave as `LocalDate`